staging/src/k8s.io/code-generator/generate-groups.sh

```diff
@@ -47,10 +47,13 @@ GROUPS_WITH_VERSIONS="$4"
 shift 4
 
 (
-  # To support running this script from anywhere, we have to first cd into this directory
-  # so we can install the tools.
-  cd "$(dirname "${0}")"
-  go install ./cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
+  # only rely on Go modules if explicitly opted in, else fallback to local copy.
+  if [ "${GO111MODULE}" == "on" ]; then 
```

Comment:
conditionalizing behavior on this envvar will not work well with different go versions (default behavior of the envvar is different between go versions)